### ISSUE TITLE
Replace Oculus controller models

### ISSUE THREAD

**Issue by dmarcos** [aframevr/a-painter#273]
We are using touch 1 models (shipped / sold with OG Oculus Rift). We should probably replace them with Touch 2 controllers. Probably not that many OG Rifts in use at the moment.

**Comment by msub2** [comment]
For sure. I'll take a look tonight and see if I can update the model to glTF also. In the future it would definitely be cool to support a wider range of controllers, either somewhat dynamically (position tip separately onto a model from A-Frame) or with some community-sourced custom models.

**Comment by dmarcos** [comment]
Yeah we can expand to more controllers. if we have a separate model for just the tip it will be easy to do.

**Comment by msub2** [comment]
Think we can close this now that #274 is in
